submissions/krylenger/friends/app.js

```diff
@@ -174,15 +170,10 @@ const addEventListeners = (usersData) => {
     filterCards(usersData);
   })
 
-  asideFilterMale.addEventListener("click", ({ target }) => {
+  asideFilterGenderContainer.addEventListener("click", ({ target }) => {
     filtersState.gender = target.id;
 		filterCards(usersData);
   });
```

Comment:
Use input type radio instead buttons, Put all your inputs into form and listen event input on form. It should be enough.

Reply:
do you mean convert all sort buttons into radio input?

Reply:
yes

Reply:
and I also mean that you don't need all these handlers, one is enough

Reply:
✅ Done